src/init/replace_vulnerability_detector.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# Copyright (C) 2023, Wazuh Inc.
+# Shell script that updates 'vulnerability-detector' to 'vulnerability-detection'
+# Author: Kevin Cardenas <kevincardenasmiranda9@gmail.com>
+
+isVulnerabilityDetectionInstalled()
+{
+    local OSSEC_CONFIGURATION_FILE="$1"
+    local VULNERABILITY_DETECTION_PATTERN="<vulnerability-detection>"
+
+    if ( grep -q "$VULNERABILITY_DETECTION_PATTERN" "$OSSEC_CONFIGURATION_FILE" ); then
+        return 0
+    fi
+
+    return 1
+}
```

Comment:
I haven't tested this yet, but I think that having no `<vulnerability-detection>` would be a valid configuration (vulnerability detection disabled), in that case, this would enable it on every upgrade, right?
If that is the case, maybe flipping the logic might work. If the config has `vulnerability-detector` fix the config.
What do you think?

Reply:
Tested with empty configuration block and module is enabled:
```
2023/12/12 10:19:41 wazuh-modulesd:vulnerability-detector: INFO: (5431): Starting vulnerability scan.
```
So default value for the `<enable>` tag is yes

----

Tested with no configuration block and module is disabled

Reply:
done